WordPress/Classes/ViewRelated/Media/MediaLibraryPickerDataSource.m

```diff
@@ -532,6 +532,53 @@ - (WPMediaRequestID)imageWithSize:(CGSize)size completionHandler:(WPMediaImageBl
     return [self.mediaID intValue];
 }
 
+- (WPMediaRequestID)videoAssetWithCompletionHandler:(WPMediaAssetBlock)completionHandler
+{
```

Comment:
Are you referring to internal method comments or method comment? I you refer to method comments, this method is the implementation of a protocol that has the associated documentation.